DR docs command correction and clarification (#2680)
hyperledger-labs/splice: docs/src/validator_operator/validator_disaster_recovery.rst

   .. code::

      val syncId = participant.synchronizers.list_connected().head.synchronizerId

>       participant.topology.party_to_participant_mappings.list(syncId, filterParticipant = <namespace>)

Reviewer: What is the context for this change?

We're interested in potentially multiple participants here, so `filterNamespace` seems like the right pick?

      val participantId = participant.id // ID of the new participant
      participant.topology.party_to_participant_mappings.propose(<party-id>, Seq((participantId, <participant-permission>)), store = syncId)

>    For this command a list of all parties can be found by running <party-id> can be found by first running:

Reviewer 2: ```suggestion
   You can list all parties that are currently on the old node by running:
```

   .. code::

      val parties = participant.parties.list (<participantIdentifier>)
  

>    Then you can map the party to a variable using a command asa such where <index> is the 0 based index of the party in the list:

Reviewer: sentence looks broken
